### PR TITLE
scan: implement top part of validatorindexpage (mobile)

### LINKED ISSUE #2265: Validator Index Page (Top Part) (mobile)

### PR DESCRIPTION
**Pull request by tansawit**
Fixed: #2265 

## Implementation details

Implement top part of `ValidatorIndexPage` mobile version

## Please ensure the following requirements are met before submitting a pull request:

- [x] The pull request is targeted against the correct target branch
- [x] The pull request is linked to an issue with appropriate discussion and an accepted design OR is linked to a spec that describes the work.
- [x] The pull request includes a description of the implementation/work done in detail.
- [x] The pull request includes any and all appropriate unit/integration tests
- [x] You have added a relevant changelog entry to `CHANGELOG_UNRELEASED.md`
- [x] You have re-reviewed the files affected by the pull request (e.g. using the `Files changed` tab in the Github PR explorer)

## If this PR doesn't need to be merged into v1.0.6 please remove this option:

- [ ] The pull request has been cherry-picked to `rc/1.0.6`

### CODE REVIEW

File: scan/src/pages/ValidatorIndexPage.re [hunk 1]
```diff
@@ -54,7 +47,26 @@ module Styles = {
...
-  let underline = style([textDecoration(`underline), color(Colors.gray7)]);
+  let underline = style([color(Colors.bandBlue)]);
+
+  // Mobile
+
+  let validatorBoxMobile =
+    style([
+      height(`px(486)),
+      width(`px(344)),
+      borderRadius(`px(4)),
```
Comment (evilpeach) [inline scan/src/pages/ValidatorIndexPage.re]: Let's use 100 percent width.

**Review by evilpeach — CHANGES_REQUESTED** [review]
Please fix detail style also.
image

File: scan/src/pages/ValidatorIndexPage.re [hunk 2]
```diff
@@ -54,7 +47,26 @@ module Styles = {
       boxShadow(Shadow.box(~x=`zero, ~y=`px(2), ~blur=`px(8), Css.rgba(0, 0, 0, 0.08))),
     ]);
 
-  let underline = style([textDecoration(`underline), color(Colors.gray7)]);
+  let underline = style([color(Colors.bandBlue)]);
```
Comment (pzshine) [inline scan/src/pages/ValidatorIndexPage.re]: Can you change `underline` to `blueLink` or something?

**Review by pzshine — CHANGES_REQUESTED** [review]

File: scan/src/pages/ValidatorIndexPage.re [hunk 3]
```diff
@@ -54,7 +47,26 @@ module Styles = {
...
+
+  // Mobile
+
+  let validatorBoxMobile =
+    style([
+      width(`percent(100.)),
+      borderRadius(`px(4)),
+      backgroundColor(`hex("FFFFFF")),
```
Comment (evilpeach) [inline scan/src/pages/ValidatorIndexPage.re]: Colors.white

File: scan/src/pages/ValidatorIndexPage.re [hunk 4]
```diff
@@ -54,7 +47,26 @@ module Styles = {
...
+      boxShadow(
+        Shadow.box(~x=`zero, ~y=`px(2), ~blur=`px(4), ~spread=`px(0), rgba(0, 0, 0, 0.08)),
```
Comment (evilpeach) [inline scan/src/pages/ValidatorIndexPage.re]: `zero ?

File: scan/src/pages/ValidatorIndexPage.re [hunk 5]
```diff
@@ -54,7 +47,26 @@ module Styles = {
...
+      ),
+      paddingLeft(`px(12)),
+      paddingTop(`px(16)),
+      paddingBottom(`px(16)),
```
Comment (evilpeach) [inline scan/src/pages/ValidatorIndexPage.re]: padding4?

File: scan/src/pages/ValidatorIndexPage.re [hunk 6]
```diff
@@ -132,14 +173,39 @@ module Uptime = {
...
+    | _ =>
+      kvRowMobile(
+        "UPTIME",
+        VReactElement(
+          <div className=Styles.headerMobile> <LoadingCensorBar width=100 height=16 /> </div>,
+        ),
+      )
+    };
+  };
+};
```
Comment (evilpeach) [inline scan/src/pages/ValidatorIndexPage.re]: Do need to create a new one for mobile. Let's use `Uptime` module. In the `Uptime` module, I just see the duplicated title and description like "Percentage of the blocks that the validator is active for out of the last 250".

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by pzshine — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]

**Pull request merged**